networking/cidr-range-definitions.adoc

```diff
@@ -0,0 +1,30 @@
+:_content-type: ASSEMBLY
+[id="cidr-range-definitions"]
+= CIDR Range Definitions
+include::_attributes/common-attributes.adoc[]
+:context: cidr-range-definitions
+
+toc::[]
+
+You must specify non-overlapping ranges for the following CIDR ranges.
+
+[NOTE]
+====
+Machine CIDR ranges cannot be changed after creating your cluster.
+====
+
+[id="machine-cidr-description"]
+== Machine CIDR
+In the Machine CIDR field, you must specify the IP address range for machines or cluster nodes, which must encompass all CIDR address ranges for your virtual private cloud (VPC) subnets. Subnets must be contiguous. A minimum IP address range of 128 addresses, using the subnet prefix `/25`, is supported for single availability zone deployments. A minimum address range of 256 addresses, using the subnet prefix `/24`, is supported for deployments that use multiple availability zones. The default is `10.0.0.0/16`. This range must not conflict with any connected networks.
+
+[id="service-cidr-description"]
+== Service CIDR
+In the Service CIDR field, you must specify the IP address range for services. The range must be large enough to accommodate your workload. The address block must not overlap with any external service accessed from within the cluster. The default is `172.30.0.0/16`. It is recommended that they are the same between clusters.
```

Comment:
Both ISG and RH Supplemental recommend not using recommend.

https://redhat-documentation.github.io/supplementary-style-guide/#introduction